sdo/data_type_sdo_context.jsonld

```diff
@@ -9,10 +9,10 @@
       "@id": "sdo:Property"
     },
     "platform": {
-       "@id": "sdo:Property"
+      "@id": "sdo:Property"
```

Comment:
no @type. Is that on purpose?

Reply:
Hi @cmadjar, this is from the original DATS SDO context files ([see here](https://github.com/datatagsuite/context/blob/master/sdo/data_type_sdo_context.jsonld#L11)). When DATS terms were already mapped in the original context, I kept them as is (i.e. I didn't overwrite or extend them). I find `sdo:Property` odd here as well, since it is technically not a property but a class instance (unlike all other property mappings). That said, I think not giving `@type` can make sense when there are many possible options for the `range` of a property. In this case, DATS defines `"platform"` as: 

> The set of instruments, software and reagents that are needed to generated the data.

([see here](https://github.com/datatagsuite/schema/blob/master/data_type_schema.json#L38))

So while it would be plausible to give for example `sdo:Text` as `@type` here, the schema also allows more complex instances of `annotation_schema`. 

I have generally not given `@type` where the range is either very broad on the DATS side of things, or on the schema.org side of things. Happy to discuss benefits / disadvantages of that decision. Maybe @zxenia also has some thoughts on this.

Reply:
ok. Good to know. Thanks for the clarification. All good then :)